operator/src/main/java/oracle/kubernetes/weblogic/domain/model/PersistentVolumeClaim.java

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2023, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.weblogic.domain.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import oracle.kubernetes.json.Description;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+public class PersistentVolumeClaim {
+
+  /**
+   * Standard object's metadata. More info:
+   * More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+   */
+  @SuppressWarnings("common-java:DuplicatedBlocks")
+  @SerializedName("metadata")
+  @Expose
+  @Valid
+  @Description("The PersistentVolumeClaim metadata. Must include the `name` field. Required.")
+  @NotNull
+  private V1ObjectMeta metadata;
+
+  /**
+   * PersistentVolumeClaimSpec is a description of a persistent volume claim.
+   */
+  @SerializedName("spec")
+  @Expose
+  @Valid
+  @Description("The specification of a persistent volume claim for `Domain in PV` domain. Required."
+      + " This section provides a subset of fields in standard Kubernetes PersistentVolumeClaim specifications."
+      + " More info: https://kubernetes.io/docs/concepts/storage/persistent-volumes#persistentvolumeclaims")
```

Comment:
If it is a subset, should we still point to standard k8s spec?

Reply:
Fixed.